spec/models/comment_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'associations' do
+    it 'belongs to user' do
+      association = described_class.reflect_on_association(:user)
+      expect(association.macro).to eq(:belongs_to)
+    end
+  end
+
+  describe 'validations' do
+    it 'validates presence of body' do
+      comment = Comment.new
+      expect(comment).not_to be_valid
+    end
```

Comment:
As i can see, there is spec for invalid, are we missing valid spec?

Reply:
Bhaiya, a comment can be either empty or not. If it is empty, it is invalid. If it is not invalid, then there must be some characters in the body. Should I check the other one too?

Reply:
@mai-wtag IMHO you should write test for both valid and invalid. Thank you

Reply:
done bhaiya